docs/Proficiency Exams/PHYS211.md

```diff
@@ -0,0 +1,55 @@
+# PHYS211
+
+[PHYS211 (University Physics: Mechanics)](../Course%20Wiki/CS%20Course%20Offerings/PHYS211.md) is the first required physics course in the Electrical and Computer Engineering curriculum, and covers the fundementals of calculus based mechanical physics.
+
+## Who Should Take the Exam
+
+If you have a solid bakcground in Physics Mechanics at the level of at least AP Physics C, then you should consider taking the proficiency exam for PHYS221. ECE students (and most engineering students) typically take PHYS211 within their first two semesters, with PHYS211 being a prerequisite for [PHYS212 (University Physics: Elec & Mag)](../Course%20Wiki/CS%20Course%20Offerings/PHYS212.md) and a requirement for Junior Eligibility.  
+
+You can still take PHYS211 even if you pass the proficiency exam, and there is no penalty for failing the proficiency exam.
+
+## Content Convered
+
+The content of the exams has typically been consistent with the contents of the final exam for this course. You can also look at the course website of the current or past semester for a list of topics in the schedule tab. Though the topics have been very consistent over semesters.
+
+- 1-D Kinematics
+- Vectors and 2-D Kinematics
+- Relative and Circular Motion
+- Newton's Laws
+- Forces and Free Body Diagrams
+- Friction
+- Work and Kinetic Energy
+- Conservative Forces
+- Work and Potential Energy
+- Center of Mass
+- Conservation of Momentum
+- Elastic Collisions
+- Impulse and Reference Frames
+- Rotational Kinematics
+- Parallel Axis Theorem 
+- Torque
+- Rotational Dynamics
+- Statics
+- Angular Momentum
+- Simple Harmonmic Motion
+- Simple and Physical Pendula
+- Harmonic Waves
+- Waves and Superposition
+- Fluid Statics
+- Fluid Dynamics
+
+## Exam Format
+
+The exam consists of both multiple choice and fill in the blank questions given through Prairietest. Typically, you are provided a [formula sheet](https://courses.physics.illinois.edu/phys211/fa2023/formula_sheet.pdf) that can be accessed through the prairietest exam. The exam may cover all content in the course and is also used as a makeup final exam for students that have an IC in the class from the previous semester.
+
+## How to Prepare
+
+The profiency exam (in recent past) has been similar to the final exam for this course. Focus on making sure you understand the fundementals of the concepts and use the practice exams avaiable through the previous semester's course website. 
```

Comment:
typo on "proficiency", "available".  Change "fundamentals of the concepts" to "fundamental concepts"